packages/web3-providers-http/package.json

```diff
@@ -25,9 +25,11 @@
 		"test:integration": "jest --config=./test/integration/jest.config.js"
 	},
 	"devDependencies": {
+		"@types/chai": "^4.3.0",
 		"@types/jest": "^27.0.3",
 		"@typescript-eslint/eslint-plugin": "^5.4.0",
 		"@typescript-eslint/parser": "^5.4.0",
+		"chai": "^4.3.6",
```

Comment:
We are using Jest in `4.x` branch and we should use it for the integration tests as well.

Reply:
fixed